.github/workflows/awstest.yml

```diff
@@ -10,12 +10,9 @@ on:
     types: [published]
 
 jobs:
-    Awstest:
-        runs-on: ${{ matrix.os }}
-        strategy:
-            matrix:
-                os: ['ubuntu-latest']
-                python-version: ['3.7']
+    run-awstest:
+        runs-on: ubuntu-latest
+        if: github.repository == 'nf-core/awsmegatests'
```

Comment:
Does this workflow script then only run, if it is in the repo `nf-core/awsmegatests`. Does this line need to be updated when running tests for a different pipeline?

Reply:
yes, that is then updated to the pipeline name. As these workflows are part of the pipeline template now, the cookie cutter automatically updates the names